assets/static/filters.js

```diff
@@ -64,12 +68,65 @@ function setPause() {
     $(selectors.icon).removeClass("fa-circle-o-notch fa-spin fa-search").addClass("fa-pause");
 }
 
+function renderHistory() {
+    const storage = window.localStorage;
```

Comment:
well, I could, but then I need to pass it through all the layers and mock it eventually anyway, so it doesn't really help much here IMHO, how strongly do you feel about it?